.github/workflows/release-plz.yml

```diff
@@ -0,0 +1,31 @@
+# Release-plz pipeline for publishing
+# a new GitHub + crates.io release
+
+name: Release-plz
+
+permissions:
+  pull-requests: write
+  contents: write
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
I don't think we have a `main` branch currently. Do you plan to rename the `master` branch?